<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCubeRTC</class>
 <widget class="QMainWindow" name="GameCubeRTC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>433</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>356</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>356</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameCube RTC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #274050;
}

QTableView {
	border: 4px solid DodgerBlue;
}

QHeaderView::section {
	border: none;
	background-color: #274050;
	padding-top: 3px;
	padding-bottom: 3px;
	color: white;
}

QLineEdit {
	qproperty-alignment: AlignCenter;
	background-color: #1D303C;
	border:none;
	border-radius: 6px;
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QLabel {
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QPushButton {
	border: none;
	border-radius: 6px;
	color: white;
	background-color: DodgerBlue;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBox" name="textBoxStartSeed"/>
    </item>
    <item row="0" column="1">
     <widget class="QTextBox" name="textBoxEndSeed"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelStartSeed">
      <property name="text">
       <string>Seed at 1/1/2000 at 00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelEndSeed">
      <property name="text">
       <string>Target Seed</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextBox" name="textBoxMinFrame">
      <property name="text">
       <string notr="true">500000</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextBox" name="textBoxMaxFrame">
      <property name="text">
       <string notr="true">1000000</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelMinFrame">
      <property name="text">
       <string>Min Frame</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelMaxFrame">
      <property name="text">
       <string>Max Frame</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonSearch">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="TableView" name="tableViewGenerator">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>356</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/QTextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textBoxStartSeed</tabstop>
  <tabstop>textBoxEndSeed</tabstop>
  <tabstop>textBoxMinFrame</tabstop>
  <tabstop>textBoxMaxFrame</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>tableViewGenerator</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
